File: Python Word Examples/WatermarksApiDemo/deleteWatermark.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.WatermarksApi(configuration)

file_name = 'ImgWatermark.docx'
#Just moving "'password': '123'" if there is no password.
extra = {'folder': "input", 'password': '123', 'dest_file_path': 'output/DeleteImgWatermarkPw.docx'}
result = api.delete_watermark(file_name, **extra)

File: JavaScript Word Examples/WatermarksApiDemo/deleteWatermark.js
(function (){
    var appId = "your id";
    var appKey = "your key";  
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.WatermarksApi(apiClient);

    var name = "TextWatermark.doc";
    var destFilePath = "output/DeleteTextWatermark_out.doc";
    var opts = {
        folder: 'input',
        storage: '',
        password: ''
    };
    instance.deleteWatermark(name, destFilePath, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();